app/legacy/v2/notifications/route_schema.py

```diff
@@ -303,6 +361,31 @@ def phone_number_or_recipient_id(self) -> Self:
             raise ValueError('You must specify at least one of phone_number or recipient identifier.')
         return self
 
+    async def get_reply_to_text(self) -> str:
+        """Get the reply_to_text field for this request.
+
+        Returns:
+            str: The reply to string
+        """
+        # Comes from sms_sender_id in the request or the Service. service_sms_sender.sender
```

Comment:
Not a big deal, but would be nice to track there's a ticket for this already.
```suggestion
        # TODO: #272 - Comes from sms_sender_id in the request or the Service. service_sms_sender.sender
```